### DISCUSSION TITLE
Cryptocurrency withdrawal fees

### DISCUSSION THREAD

**Discussion by NavBlue1** [ghostfolio/ghostfolio#3923] [Q&A]
Is there any way to input cryptocurrency exchange withdrawal fees?

When withdrawing cryptocurrency from an exchange the withdrawal fee is usually charged in the cryptocurrency that is being withdrawn.  

I can see where to create a Fee activity, however when doing so the fee currency is my base currency & not the cryptocurrency asset.

Am I better off just reducing the Quantity of a cryptocurrency Buy activity equal  to my total withdrawal fees account for this?

**Comment by swissbuechi** [comment]
> Am I better off just reducing the Quantity of a cryptocurrency Buy activity equal to my total withdrawal fees account for this?

@NavBlue1 I usually just create a sell action with the matching quantity and amount. No edits needed this way.
